python/ql/src/Security/CVE-2018-1281/BindToAllInterfaces.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>One binds a socket to a given address and a given port
+to make it possible to receive data on this particular socket.
+Alternatively, this can also be done to use the (IP address, port) pair
+as the source of the data being sent.
+To bind a socket to all available IPv4 addresses
+when using the built-in Python <code>socket</code> module
+(for instance, when building a message sender service
+or an FTP server data transmitter),
+one can specify <code>0.0.0.0</code> as the IP address.
+When you bind the port to all interfaces,
+you essentially allow to accept connections from any IPv4 address
+provided that it can get to the socket via routing.
+Binding to all interfaces is therefore associated with security risks.</p>
+</overview>
+
+<recommendation>
+<p>Bind your service incoming traffic only to a dedicated interface.
+If you think you would need to bind more than one interface
```

Comment:
```suggestion
If you think you need to bind more than one interface
```

Reply:
God, why do I like using would everywhere... Maybe it's some gestalt I hadn't closed properly since a 7th grade English class when I got F...